Game.py

```diff
@@ -0,0 +1,49 @@
+import pygame
+import sys
+import pygame.freetype
+from Main_menu import MainMenu
+
+red = (255, 0, 0)
+green = (0, 255, 0)
+blue = (0, 0, 255)
+yellow = (255, 255, 0)
+black = (0, 0, 0)
+white = (255, 255, 255)
+
+
+class Game(object):
+    def __init__(self):
+        self.max_tps = 300.0
+        self.tps_delta = 0.0
+        # Init
+        pygame.init()
+        self.screen = pygame.display.set_mode((1280, 720))
+        self.tps_clock = pygame.time.Clock()
```

Comment:
Konwencja w pythonie,  _ protected, __ private (https://www.tutorialsteacher.com/python/private-and-protected-access-modifiers-in-python). Zmienne które nie będą używane poza klasą powinny być private i mieć __ na początku nazwy